src/wrappers/responsiveContainingView/component.tsx

```diff
@@ -14,6 +14,7 @@ interface IStyledResponsiveContainingViewProps extends IWrapperProps {
   size: ResponsiveField<number>;
   isFullWidth: boolean;
   shouldIncludeMaxSize: boolean;
+  centerHorizontally: boolean;
```

Comment:
please copy variable name from ContainingView. I know its a bit longer but its good practice to have rules for boolean names